tools/wptrunner/wptrunner/browsers/sauce.py

```diff
@@ -127,15 +127,29 @@ def get_tar(url, dest):
 class SauceConnect():
 
     def __init__(self, **kwargs):
-        self.config = kwargs["config"]
         self.sauce_user = kwargs["sauce_user"]
         self.sauce_key = kwargs["sauce_key"]
         self.sauce_tunnel_id = kwargs["sauce_tunnel_id"]
         self.sauce_connect_binary = kwargs.get("sauce_connect_binary")
         self.sc_process = None
         self.temp_dir = None
+        self.env_config = None
+
+    def __call__(self, env_options, env_config):
+        self.env_config = env_config
+
+        return self
+
+    def __enter__(self):
+        # Because this class implements the context manager protocol, it is
```

Comment:
You just need to change the comment and error message to account for the fact that `use` doesn't exist.